webpack.config.js

```diff
@@ -36,5 +38,23 @@ const config = {
       },
     ],
   },
+  node: {
+    __dirname: false,
+  },
+  plugins: [
+    new CopyWebpackPlugin({
```

Comment:
Please add comments for what it is doing

Reply:
It's documented here for users of `cloud-mta`:
https://github.com/SAP/cloud-mta#packaging-with-webpack
Referenced from `mta-lib`:
https://www.npmjs.com/package/@sap/mta-lib (see "Packaging with webpack" section)
I didn't want to copy the documentation.

Reply:
I will add a link to the documentation in the code